clang/tools/libclang/CIndex.cpp

```diff
@@ -8648,6 +8646,119 @@ void clang_annotateTokens(CXTranslationUnit TU, CXToken *Tokens,
   }
 }
 
+//===----------------------------------------------------------------------===//
+// Operations for querying information of a GCC inline assembly block under a
+// cursor.
+//===----------------------------------------------------------------------===//
+CXString clang_Cursor_getGCCAssemblyTemplate(CXCursor Cursor) {
+  if (!clang_isStatement(Cursor.kind))
+    return cxstring::createEmpty();
+  if (auto const *Stmt = dyn_cast_or_null<GCCAsmStmt>(getCursorStmt(Cursor))) {
```

Comment:
`Stmt` is a type name, better to pick a name which doesn't conflict with a type (this helps folks who have editors that don't handle conflicts with type and object names for things like autocomplete). Same issue happens in a bunch of other functions as well.